internal/server/data/group_test.go

```diff
@@ -52,7 +58,7 @@ func TestCreateGroupDuplicate(t *testing.T) {
 	)
 
 	db := setup(t)
-	createGroups(t, db, everyone, engineers, product)
+	createGroups(t, db, &everyone, &engineers, &product)
```

Comment:
I changed these to pointers so that we could get access to the ID which is generated on save. The ID is often needed for other queries.